src/Assignments/ProblemIframe.tsx

```diff
@@ -83,13 +83,7 @@ export const ProblemIframe: React.FC<ProblemIframeProps> = ({
             if (currentReq.cancelled) {
                 logger.debug(`Problem Iframe: The request for problem #${problem.id} and workbook #${workbookId} was cancelled early.`);
                 return;
-            } else if (_.isNil(rendererHTML)) {
-                // Preview Problems won't return a rendererHTML when the path is bad.
-                if (_.isNil(previewPath) && _.isNil(previewProblemSource)) {
-                    logger.error(`Problem Iframe: Request for problem #${problem.id} came back null`);
```

Comment:
If appropriate this would have already been logged here:
https://github.com/rederly/frontend/blob/eb44b14d95def7cb744ca9b77757e29160833c21/src/Assignments/ProblemIframe.tsx#L143